internal/pkg/deploy/cloudformation/stack/testdata/workloads/static-site.stack.yml

```diff
@@ -211,3 +211,24 @@ Resources:
                 Resource: !Sub arn:aws:s3:::${Bucket}/*
       ManagedPolicyArns:
         - !Sub arn:${AWS::Partition}:iam::aws:policy/service-role/AWSLambdaBasicExecutionRole
+
+  EnvManagerS3Access:
+    Metadata:
+      aws:copilot:description: A policy that gives the Env Manager role access to this site's S3 Bucket
+    Type: AWS::IAM::Policy
+    Properties:
+      Roles:
+        - !Sub "${AppName}-${EnvName}-EnvManagerRole"
+      PolicyName: !Sub "${WorkloadName}-S3Access"
```

Comment:
Does the policy name have to be unique in a region / globally?

Reply:
it needs to be unique in a region since it's attached to the `EnvManagerRole`! Separate regions have their own EnvManagerRole, so I think this should be fine.

Reply:
`EnvMangerRole` is per-env though! It's still possible for a region to have multiple envs.